client/elements/menus/sc-language-base-menu-css.js

```diff
@@ -1,85 +1,101 @@
 import { html } from 'lit-element';
 
 export const languageBaseMenuCss = html`
-<style>
-  :host {
-    --primary-color: var(--sc-primary-color);
-    --paper-menu-button-content: {
-      display: block;
-    };
-  }
+  <style>
+    :host {
+      overflow-y: scroll;
+      --mdc-menu-min-width: 500px;
+      --mdc-theme-text-primary-on-background: var(--sc-primary-text-color);
+    }
+
+    .separator {
+      background-color: var(--sc-border-color);
+      width: 100%;
+      overflow: hidden;
+      height: 1px;
+      margin-top: var(--sc-size-xxs);
+      margin-bottom: var(--sc-size-xxs);
+    }
+
+    .language-menu-dropdown {
+      font-size: var(--sc-skolar-font-size-md);
+      background-color: transparent;
+      --paper-input-container-focus-color: var(--sc-primary-accent-color);
+      --paper-dropdown-menu-icon: {
+        color: var(--sc-disabled-text-color);
+      }
 
-  .language-menu-dropdown {
-    font-size: var(--sc-skolar-font-size-md);
-    background-color: transparent;
-    --paper-input-container-focus-color: var(--sc-primary-accent-color);
-    --paper-dropdown-menu-icon: {
-      color: var(--sc-disabled-text-color);
-    };
-    
       --paper-input-container-input-color: var(--sc-primary-text-color);
       --paper-input-container-color: var(--sc-secondary-text-color);
 
-    --paper-menu-button-dropdown: {
-      box-shadow: var(--sc-shadow-elevation-12dp);
-      width: 180px;
+      --paper-menu-button-dropdown: {
+        box-shadow: var(--sc-shadow-elevation-12dp);
+        width: 180px;
+        background-color: var(--sc-secondary-background-color);
+      }
+    }
+
+    .language-menu-list {
       background-color: var(--sc-secondary-background-color);
-    };
-  }
+    }
 
-  .language-menu-list {
-    background-color: var(--sc-secondary-background-color);
-  }
-  
-  .language-menu-paper-item {
-    font-size: var(--sc-skolar-font-size-md);
-    color: var(--sc-primary-text-color); 
-    /*19px for the icon, 16px for the margin */
-    --paper-item-icon-width: calc(var(--sc-size-language-icon) + var(--sc-size-md));
-  }
+    .language-menu-paper-item {
+      font-size: var(--sc-skolar-font-size-md);
+      color: var(--sc-primary-text-color);
+      /*19px for the icon, 16px for the margin */
```

Comment:
this comment is likely to be totally inaccurate, since those are vars and not absolute values

Reply:
I'm wondering now if this css is still need because it was for paper item and which as I remember we don't use anymore.

Reply:
Removed in 07643a2f